### ISSUE TITLE
What happened to the 2ndquadrant/pgaudit repository???

### ISSUE THREAD

**Issue by ivoras** [pgaudit/pgaudit#28]
Hello,

I've noticed that the 2ndquadrant/pgaudit repository was apparently destroyed and re-forked from this one. I realise this question is about a different repo from this one, but it's kind of important.

The problem is that the 2ndquadrant/pgaudit code was compatible with PostgreSQL 9.3, which is both officially supported by postgresql.org and (somewhat unfortunately), still used by an organisation I'm working for, which has planned to deploy the (old) pgaudit on.

Basically, my question is: does anyone here still have a copy of the old repo?

**Comment by dwsteele** [comment]
Hello Ivan,

I think it would be best to contact 2nd Quadrant directly about getting older versions of their pgaudit module.

Regards,
-David

**Comment by ivoras** [comment]
In case anyone in the future is interested, I have a fork of the old version (for pg 9.3) working here: https://github.com/ivoras/pgaudit

**Comment by dwsteele** [comment]
Thanks, that's helpful for archival purposes at the very least!

**Comment by simonat2ndQuadrant** [comment]
The pgaudit extension was originally written by Simon Riggs and Abhijit Menon-Sen of 2ndQuadrant as part of the AXLE project for the EU. The next version was designed jointly by Simon Riggs and David Steele to provide object level logging. The original version has been deprecated and is no longer available.
